### DISCUSSION TITLE
asuswrt merlin 3004.388.7 firmware

### DISCUSSION THREAD

**Discussion by CHazz** [Vaskivskyi/ha-asusrouter#816] [General]
hi, are you planning support for asuswrt merlin 3004.388.7 firmware?
I'm beta testing but the addon can't be used.

thank you very much for the good work

**Comment by Vaskivskyi** [comment]
Hello. Yes, 388.7 will be supported as soon as possible. Sorry for the delay in the reply. You can watch issues #817 or #818 for the status (both on the same matter)
